sdk/identity/azure-identity/checkstyle-suppressions.xml

```diff
@@ -60,6 +60,7 @@
   <suppress files="com.azure.identity.implementation.HttpPipelineAdapter.java" checks="com.azure.tools.checkstyle.checks.EnforceFinalFieldsCheck" />
   <suppress files="com.azure.identity.implementation.IdentityLogOptionsImpl.java" checks="com.azure.tools.checkstyle.checks.EnforceFinalFieldsCheck" />
   <suppress files="com.azure.identity.implementation.IntelliJAuthMethodDetails.java" checks="com.azure.tools.checkstyle.checks.EnforceFinalFieldsCheck" />
+  <suppress files="com.azure.identity.implementation.IntelliJAuthMethodDetails.java" checks="com.azure.tools.checkstyle.checks.EnforceFinalFieldsCheck" />
```

Comment:
Do we need this suppression in v1 azure-identity? There are no changes to any source code in v1.

Reply:
That's a duplicate, accidental addition, will be removed.